tests/unittest/tools/test_test_to_stage_mapping.py

```
@@ -109,6 +110,82 @@ def test_s3_stdout_echo_requires_explicit_opt_in():
+        '"Filled-PyTorch-1": ["x", "l0_filled", 1, 1],\n'
+        '"Empty-PyTorch-1": ["x", "l0_empty", 1, 1],\n')
+    db_dir = tmp_path / 'tests' / 'integration' / 'test_lists' / 'test-db'
+    db_dir.mkdir(parents=True)
+    for name, stage in (('l0_filled', 'pre_merge'), ('l0_empty', 'post_merge')):
+        (db_dir / f'{name}.yml').write_text(
+            f'version: 0.0.1\n{name}:\n- condition:\n    terms:\n'
+            f'      stage: {stage}\n      backend: pytorch\n'
+            f'  tests:\n  - unittest/{name}.py\n')
+
+    script = os.path.join(SCRIPTS_DIR, 'test_to_stage_mapping.py')
+    proc = subprocess.run([
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)